doc/examples/build.bat

```diff
@@ -0,0 +1,8 @@
+cc -O3 -fomit-frame-pointer -I../.. -o libco.o -c ../../libco.c
+c++ -O3 -fomit-frame-pointer -I../.. -c test_timing.cpp
+c++ -O3 -fomit-frame-pointer -o test_timing libco.o test_timing.o
+c++ -O3 -fomit-frame-pointer -I../.. -c test_args.cpp
+c++ -O3 -fomit-frame-pointer -o test_args libco.o test_args.o
+c++ -O3 -fomit-frame-pointer -I../.. -c test_serialization.cpp
+c++ -O3 -fomit-frame-pointer -o test_serialization libco.o test_serialization.o
```

Comment:
MSVC (`cl.exe`) invocations and flags should be added too at some point (not necessarily now), since most people won't be using MinGW/Clang? Perhaps something like the following:

```
WHERE cl.exe >nul 2>&1
IF %ERRORLEVEL% EQU 0 (
    REM run commands here
)
```

Reply:
I'm definitely open to adding an MSVC build-script (maybe rename the current one to `build-mingw.bat` and make a new `build-msvc.bat`), but that can be done by anybody later, it doesn't need to be done in this PR in order to be useful.